src/test/regress/expected/failure_single_mod.out

```diff
@@ -44,7 +44,7 @@ WHERE shardid IN (
 ) AND shardstate = 3 RETURNING placementid;
  placementid 
 -------------
-         137
+         125
```

Comment:
can this be made deterministic? / is the placementid relevant?

Reply:
> / is the placementid relevant?

I think not at all, it is used to show that there is one placement with `shardstate = 3 `.

I'm currently having some issues on my local machine with pipenv/python versions, so couldn't run the failure tests locally and fix this. I'll do it today/tomorrow, merging the PR for now